### ISSUE TITLE
Raspberry Pi 3 with Motioneye OS not Detecting Pi Camera

### ISSUE THREAD

**Issue by wenzelp1** [ccrisan/motioneye#1079]
Hi All- I've got a Raspberry Pi 3 with the Pi Camera v2.  I'm running MotionEye OS but after it configures and I login to the web dashboard the camera is not displayed.  This may be a basic question but I've triple checked my work, reinstalled the OS just to be sure and checked to ensure the camera works properly on Raspbian.   Still no luck.  It's very frustrating as surely I'm missing something simple; can anyone help?

Thank you!!

<img width="1072" alt="dashboard" src="https://user-images.githubusercontent.com/45518048/49330769-0172df00-f559-11e8-96d2-65f49ded3bfc.PNG">
![piscreenshot](https://user-images.githubusercontent.com/45518048/49330770-0172df00-f559-11e8-9d63-907ba7413f5f.jpg)
![pisetup](https://user-images.githubusercontent.com/45518048/49330771-0172df00-f559-11e8-9c02-14decb6e9dfa.jpg)

**Comment by ccrisan** [comment]
Does your camera work with Raspbian?

**Comment by wenzelp1** [comment]
thanks for the follow-up, yes, I've confirmed it works on Raspbian

**Comment by elmicha** [comment]
Did you disconnect the camera cable between using Raspbian and Motioneye OS? If so, did you make sure the orientation of the cable contacts is correct on both ends?

This might sound like I would suggest that you're an idiot, but in fact I have done exactly that a few days ago. Disconnected the cable, reconnected it and paid absolutely no attention to the orientation of the cable. It took over an hour for me to realize that oh, that cable has two sides, and only one of them has connectors.

**Comment by wenzelp1** [comment]
thanks for the feedback elmicha, it's a very fair question to ask :-).  I have double checked that early on and it's correct.  Am I missing something in the setup?  Flash the SD card with motioneye os, fire up the Pi with the camera connected and an ethernet cable plugged in, let it configure / boot up, and then log into the motioneye dashboard from a seperate PC using the Pi's IP address?

**Comment by wenzelp1** [comment]
bump

massively appreciate the help!

**Comment by elmicha** [comment]
No, what you did sounds correct. You can't have the same camera as MMAL and V4L2 camera at the same time, but you probably already tried each on its own. If you enable ssh, login with ssh and type `dmesg`, I guess you can see that the camera is not recognized?
